albam/engines/mtfw/mesh.py

```diff
@@ -1101,13 +1095,15 @@ def _serialize_groups(src_mod, dst_mod):
         src_group = src_mod.groups[i]
```

Comment:
Would it make sense to make this a custom property?

Reply:
According to the structure, they are just spheres with position and radius and I am kinda confused, are they an approximate volume for meshes in the group? 3ds max script that overwrites them often breaks motion blur.

Reply:
oh, OK, I thought they might be related to visibility, some modders had trouble with this in cut-scenes, but I never knew what this was for.

Reply:
Well I checked and it looks like group IDs and render group param are linked but I still don't understand why those groups are defined by spheres, need to re-create them in Blender, maybe it will make sense then